web/src/pages/day/[dayIndex].tsx

```diff
@@ -146,6 +149,7 @@ const DayPage: NextPage<DayPageProps> = ({
       }
 
       setInputMode(value);
+      setShowWrongFeedbackMessage(false);
```

Comment:
What do you think about hiding the wrong answer feedback message whenever the user starts typing in any of the fields?

Reply:
Yes, that is better!